manifests/4.4/cluster-kube-descheduler-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,233 @@
+
+
+      ## Parameters
+      The Descheduler operator exposes the following parameters in its CRD:
+
+      * `deschedulingIntervalSeconds` - this sets the number of seconds between descheduler runs
+      * `image` - specifies the Descheduler container image to deploy
+      * `flags` - this allows additional descheduler flags to be set, and they will be appended to the descheduler pod. Therefore, they must be in the same format as would be passed to the descheduler binary (eg, `"--dry-run"`)
+  displayName: Descheduler Operator
+  keywords: ["scheduling", "descheduler", "workload"]
+  provider:
+    name: Red Hat, Inc.
+  maturity: alpha # or beta/stable maybe?
+  version: 4.4.0
+  links:
+  - name: Source Code
+    url: https://github.com/openshift/cluster-kube-descheduler-operator
+  maintainers:
+  - email: support@redhat.com
+    name: Red Hat
+  minKubeVersion: v1.16.0 # or lower?
+  labels:
+    olm-owner-enterprise-app: cluster-kube-descheduler-operator
+    olm-status-descriptors: cluster-kube-descheduler-operator.v4.4.0
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  install:
+    spec:
+      clusterPermissions:
+      - serviceAccountName: openshift-descheduler
+        rules:
+        - apiGroups:
+          - operator.openshift.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - kubedeschedulers.operator.openshift.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - pods
+          - configmaps
+          - secrets
+          - names
+          - nodes
+          - pods/eviction
+          - events
+          verbs:
+          - "*"
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - replicasets
+          verbs:
+          - "*"
+        - apiGroups: ["batch", "extensions"]
+          resources: ["jobs", "cronjobs"]
+          verbs:
+          - "*"
+      deployments:
+      - name: descheduler-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: descheduler-operator
+          template:
+            metadata:
+              labels:
+                name: descheduler-operator
+            spec:
+              containers:
+                - name: descheduler-operator
+                  image: docker.io/mdame/cluster-kube-descheduler-operator
```

Comment:
That needs changing, right?

Reply:
Right, just found https://quay.io/repository/openshift/origin-descheduler?tab=tags. Will update csv and retest.